plugins/generic/pln/templates/ping.tpl

```diff
@@ -9,6 +9,11 @@
 		<releaseDate>{$pluginVersion.date}</releaseDate>
 		<installed>{$pluginVersion.version->getDateInstalled()}</installed>
 		<current>{$pluginVersion.version->getCurrent()}</current>
+		<terms termsAccepted="{$termsAccepted|escape}">
+			{iterate from=termsDisplay item=term}
+			<term key='{$term.key}' updated='{$term.updated}' accepted='{$term.accepted}'>{$term.term|escape}</term>
```

Comment:
Use " for attribute quoting, and pass content through the `|escape` modifier. (Chances are there isn't a leak risk here, but there's no harm done.)